pulp_ansible/tests/functional/utils.py

```diff
@@ -163,6 +167,52 @@ def wait_tasks():
         running_tasks = tasks.list(state="running")
 
 
+def gen_collection_in_distribution(
+    base_path, name=None, namespace=None, versions=None, **cfg_kwargs
+):
+    """
+    Generate a randomized collection and upload it to the specified repository.
+
+    Params:
```

Comment:
This isn't a big deal, but Pulp's docstring style all over the code is different than this. https://docs.pulpproject.org/pulpcore/contributing/style-guide.html?highlight=docstring#in-code-documentation